docs/guides/databases/mariadb/backup-mariadb-mysql-to-object-storage-with-restic/index.md

```diff
@@ -1,16 +1,18 @@
 ---
+slug: backup-mariadb-mysql-to-object-storage-with-restic
 author:
   name: Andy Heathershaw
   email: andy@andysh.uk
-description: 'This guide shows you how to use Restic to backup your MariaDB or MySQL databases onto Linode Object Storage.'
-og_description: 'Learn how to backup your MariaDB and MySQL databases off your Linode and onto Linode Object Storage with Restic.'
+description: 'Restic is a backup utility written in Go. This guide shows how to configure Restic to backup your MariaDB (or MySQL) databases onto Linode Object Storage.'
```

Comment:
I updated the description to be a bit closer to 150-160 characters, which is the guidance we've received from our SEO consulting agency about the target length for the description

This should probably be implemented as a Travis CI warning, but that's for another day :) In any case, it's not a big deal, just a tweak